boards/common/openocd.board.cmake

```diff
@@ -16,7 +16,13 @@ endif()
 set(OPENOCD_CMD_LOAD_DEFAULT "${OPENOCD_FLASH}")
 set(OPENOCD_CMD_VERIFY_DEFAULT "verify_image")
 
+# Cortex-M-specific halt debug enable bit clearing
```

Comment:
Can you add some more comments here, since this looks "weird"? Maybe just linking to the openocd PR you opened would be good enough.

Reply:
I'll comment properly.